### ISSUE TITLE
examples/tilt_shift  undefined: imagick.QUANTUM_RANGE

### ISSUE THREAD

**Issue by woniu369** [gographics/imagick#57]
```
# github.com/gographics/imagick/examples/tilt_shift
./main.go:26: undefined: imagick.QUANTUM_RANGE
```
```
# github.com/gographics/imagick/examples/gel
./main.go:66: undefined: imagick.QUANTUM_RANGE
```

**Comment by woniu369** [comment]
```
examples/clipmask
panic: ERROR_BLOB: unable to open image `tile_water.jpg': No such file or directory @ error/blob.c/OpenBlob/2645
```

**Comment by justinfx** [comment]
I don't remember the exact error this was addressing, but QUANTUM_RANGE was commented out for a reason https://github.com/gographics/imagick/commit/dec28cfe85af09b3ca7b2bc010c3db0ef5d0a99f

Looks like stragglers in the examples that weren't updated

**Comment by justinfx** [comment]
Can you give #58 a try?

**Comment by woniu369** [comment]
thank you

**Comment by justinfx** [comment]
Sure. Merged to master and 6.8.9
